tests/PHPStan/Rules/PhpDoc/data/generic-callables-incompatible.php

```diff
@@ -191,3 +191,14 @@ function shadowsParamOutArray(array &$existingClasses): void
 function shadowsReturnArray(): array
 {
 }
+
```

Comment:
Do `// lint >= 7.4` at the top of the file, see many comments like that in tests files :)

Reply:
good to know, dunno how i never noticed those before